### PR TITLE
fix: publish cjs and esm modules

### PR DESCRIPTION
**Pull request by magicmatatjahu**
**Description**

Publish `esm` and `cjs` modules to npm. Use for imports `module` and `main` fields in package.json. Publish also types for TS.

**Related issue(s)**
Resolves https://github.com/asyncapi/modelina/issues/228

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,6 +38,7 @@ This package is under development and it has not reached version 1.0.0 yet, whic
 - [Customization](#customization)
 - [Development](#development)
 - [Contributing](#contributing)
+- [Contributors ✨](#contributors-%E2%9C%A8)
```
Comment (jonaslagoni) [inline README.md:41]: What is this? 😆

Comment (magicmatatjahu) [reply README.md:41]: Generated by `markdown-toc` 😄

**Review by jonaslagoni — APPROVED** [review]
👍

**Comment by magicmatatjahu** [comment]
@jonaslagoni Thanks!

**Pull request merged**